index.js

```diff
@@ -27,16 +27,15 @@ expressApp.get('/_ah/start', (req, res) => {
 expressApp.use(bodyParser.json(), app);
 
 // Start server
-expressApp.listen(process.env.PORT || 8080);
-
+expressApp.listen(process.env.PORT || 8080, () => console.log('Listening at port 8080'));
 
 /**
  * Default Welcome Intent
  */
 app.intent('Default Welcome Intent', conv => {
   // Create a session for this user at the beginning so user
   // is logged on to Circuit by the time needed
-  if (!sessions[conv.user.id]) {
+  if (!sessions[conv.user.storage]) {
```

Comment:
Not sure I understand this change. Is this a change in the API from google assistant? What does "storage" contain?

Reply:
Yes. <b>user.id</b> was deprecated in place of <b>user.storage</b>.